site/documentation/content/api/telemetry-pubsub/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Telemetry API Specification for Google Pub/Sub"
+linkTitle: "Telemetry API (Pub/Sub)"
+weight: 405
+resources:
+- src: publish_pubsub_qos0.svg
+- src: publish_pubsub_qos1.svg
+---
+
+The *Telemetry* API is used by *Protocol Adapters* to send telemetry data downstream.
+*Business Applications* and other consumers use the API to receive data published by devices belonging to a particular
+tenant.
+
+The Telemetry API is defined by means of Google Pub/Sub message exchanges, i.e. a client needs to connect to Hono and configure
+Hono for Google Pub/Sub messaging infrastructure in order to use it and invoke operations of the API as described in the following sections. Throughout the remainder of
+this page we will simply use *Pub/Sub* when referring to Google Pub/Sub.
+
+The Telemetry API for Pub/Sub is an alternative to the [Telemetry API for Kafka]({{< relref "/api/telemetry-kafka" >}})
+for applications that want to consume telemetry data from a Pub/Sub Messaging Network instead of an Apache Kafka&trade;
+broker.
+
+## Southbound Operations
+
+The following operation can be used by *Protocol Adapters* to publish telemetry data received from devices to
+consumers like *Business Applications*.
+
+### Publish Telemetry Data
+
+The protocol adapter writes messages to the tenant-specific topic `projects/${google_project_id}/topics/${tenant_id}.telemetry`
+where `${google_project_id}` is the ID of the Google project and `${tenant_id}` is the
+ID of the tenant that the client wants to upload telemetry data for.
+
+**Preconditions**
+
+1. The Google Project ID is declared in the deployment file.
```

Comment:
I do not think that users will know what the *deployment file* refers to ...
In fact, I do not know either ;-)